### ISSUE TITLE
does not work on AMD RV100/Radeon 7000 with 32 bit Fedora

### ISSUE THREAD

**Issue by LinuxPersonEC** [imageguy/fglrx-for-Fedora#15]
using Advanced Micro Devices, Inc. [AMD/ATI] RV100 [Radeon 7000 / Radeon VE]

I get a message that the driver is not compatible.

**Comment by mirh** [comment]
Mhh.. Let's see, where could I start?
1. This is about latest ~2015 fglrx (dx11 cards only)
2. Even if it wasn't, I think the latest [remotely](https://aur.archlinux.org/packages/catalyst-total-hd234k/) still usable closed driver on new distributions is [13.1](https://web.archive.org/web/20130906020343/http://support.amd.com/us/gpudownload/linux/legacy/Pages/legacy-radeon_linux.aspx) (which is dx10 cards)
3. Further back we have [9.3](http://support.amd.com/us/gpudownload/linux/legacy/Pages/radeon_linux.aspx?type=2.4.1&product=2.4.1.3.12&lang=English), which I guess could still somehow be a workable option, should one [lower](https://github.com/aur-archive/catalyst-old) to use some 2010 distro (dx9 cards)
4. Going even fucking farther in time, we have [8.28.8](https://web.archive.org/web/20070102110348/http://ati.amd.com/support/drivers/linux/previous/linux-r-8-28-8.html).. But at this point it's utter masochism. 
5. Oh, and nonetheless even this still wouldn't support dx7-gen cards. 

So TL;DR there's just open driver available for that. 
I, for one, want to take my hats off, before such a venerable gpu.. But whatever your problem for wanting to change drivers, freedesktop's bugzilla is your place. 
(or just stick with windows, seriously)

**Comment by LinuxPersonEC** [comment]
Thanks for the reply. It's a long shot effort to see if I can get a DE/desktop working. I [have a bug at freedesktop](https://bugs.freedesktop.org/show_bug.cgi?id=102409) for months and just dead air. It's an older Dell PowerEdge 1850 used for backups connected to a VessRAID.

**Comment by mirh** [comment]
Oh, this start to sound interesting. 
Bisecting should be quite easy though, at least for starters. 
Be it kernel or xorg.

**Comment by LinuxPersonEC** [comment]
Are you suggesting I do a bisection or that the folks at freedesktop do so? The Xorg folks said it is not an org issue. And [this user](https://bugs.freedesktop.org/show_bug.cgi?id=102409#c2) says: 

> > "GLib: g_variant_new_string: assertion 'string != NULL' failed" is from gym, not Xorg, and appears to be it's response to Xorg crashing, not the cause.

**Comment by mirh** [comment]
That you do it. 
And I mean, not necessarily to the exact commit. Even just finding which major version did what would be useful.

**Comment by LinuxPersonEC** [comment]
Can you recommend a good tutorial and specifically how I would use it to test a desktop manager like GDM or KDM? I see [this example](https://fedoraproject.org/wiki/User:Ignatenkobrain/Kernel/Bisection) for Fedora but how would I use it with a DM?

**Comment by mirh** [comment]
That super-precise thing, is the latest step if any. 
For the moment.. Just figuring out what broke in the first place would be enough. And you can just use already build binaries for that. 
Just manually downgrade packages, and see when everything starts to work again.

**Comment by LinuxPersonEC** [comment]
I downgraded all of the xorg-x11-server-Xorg xorg-x11-server-common packages down to:
```
X.Org X Server 1.18.3
Release Date: 2016-04-04
X Protocol Version 11, Revision 0
Build Operating System:  4.4.9-300.fc23.x86_64
Current Operating System: Linux ourschool.edu 4.13.10-200.fc26.i686+PAE #1 SMP Fri Oct 27 16:00:10 UTC 2017 i686
```
GDM continues to coredump and no other DM starts. Do I have to downgrade the kernel along with x?

**Comment by mirh** [comment]
You should continue to try downgrading everything, until it finally starts to work.

**Comment by mirh** [comment]
@RobbieTheK You could even just try live Fedora releases backwards until you find one that is finally working, just for starters. 

Or.. just to test what newer fedora will have "ahead", I'd recommend also giving a spin to [manjaro32](https://forum.manjaro.org/t/17-1-1-rc1-testing-xfce-32-bit-installer-image/37488)
